src/main/groovy/com/rundeck/plugins/logging/JsonLogFilter.groovy

```diff
@@ -134,7 +145,11 @@ See [here](https://github.com/eiiches/jackson-jq#implementation-status-and-curre
         Integer i=0
         list.each{it->
             if(it.getNodeType() == JsonNodeType.STRING){
-                allData.put(prefix +"."+ i.toString(),it.asText())
+                def value = it.textValue()
```

Comment:
the value used by default changed from `asText()` to `textValue()`, was that intentional?

Reply:
I checked the code and it does the same 
![Screenshot 2024-02-01 at 15 31 28](https://github.com/rundeck-plugins/jq-json-logfilter/assets/6034968/04bfd3f7-4ed8-48d1-9790-b6b6f5289276)